.github/workflows/pr-snapshot-release.yaml

```diff
@@ -106,8 +106,6 @@ jobs:
         with:
           node-version: 20
```

Comment:
As an aside, I think you can always use the node version of your project by using the `node-version-file`.
https://github.com/actions/setup-node?tab=readme-ov-file#usage

Reply:
Thanks! Fixed it at b1e4bc7.